server.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('dotenv').config();
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('./db');
+const router = require('./routes/router')
```

Comment:
Just a personal preference: I think renaming `routes/router.js` to `routes/index.js` is cleaner while also exposing a `useRoutes(...)` function.

You'll then have it read:
```
const routes = require('./routes');

...

routes.attachRoutes(app);
...
```